src/libs/migrations/RenameExpensifyNewsStatus.ts

```diff
@@ -15,8 +14,9 @@ export default function () {
                 Onyx.disconnect(connectionID);
 
                 // Fail early here because there is nothing to migrate
-                if (!user || _.isNull(user.expensifyNewsStatus) || _.isUndefined(user.expensifyNewsStatus)) {
+                if (user?.expensifyNewsStatus === null || user?.expensifyNewsStatus === undefined) {
```

Comment:
```suggestion
                if (!user?.expensifyNewsStatus) {
```

We can simplify this condition